terraform/modules/deploy_hook/lambdas/hook.py

```diff
@@ -38,6 +38,7 @@ def lambda_handler(event, _context):
         put_lifecycle_event_hook(event, 'Succeeded')
     except:
         put_lifecycle_event_hook(event, 'Succeeded')
```

Comment:
Are we changing this back to Failed in another step?

Reply:
Will change to failed in a later commit